src/atoms/TextInput/Label.native.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
Why are we making this a separate file? Let's define this as a separate component inside `TextInput.native.js` file itself.

Reply:
Also, let's namespace files inside component dir. Eg. `TextInputLabel.native.js`
// @kamleshchandnani

Reply:
Not so sure about this. If we follow this pattern, we'll have Label, HelpText, AccessoryIcon, AccessoryText, ErrorText, AnimatedLabel, Line inside the same file TextInput.native.js which will then go up to ~900 lines

I've referred other DS and this is a standard practice to create components in separate files. This way it's easier to maintain them. @kamleshchandnani @shubhnik @shridhar52war thoughts?

Reply:
I'm fine with this approach since we have too much of logic in TextInput.
Also @dhruvdutt for having namespace it's too much. To solve your search problem on editor you can just add the folder name. since conflicting occurences will be very less unlike `index.js`

Reply:
@deorukhkarchaitanya what we can rather do is that group same files. for example you can put Normal Label and AnimatedLabel inside the same file and export them as as `Label.Regular` and `Label.Animated`